charts/console/CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and is generated by [Changie](https://github.com/miniscruff/changie).
 
 
+## Unreleased
+### Changed
+* Prepare console and redpanda charts for Console v3 configuration changes.
```

Comment:
These are meant to be user facing release notes. We'll want some more details here. If there's a docs page we can link to that describes the configuration changes, that'd be great! Otherwise let's include the new version range of console that's supported and call out the versions of console that are no longer supported.

It would also be good to briefly call out the changes to the values file:
- Removed roles and rolebindings
- connect -> kafkaConnect.
- schema registry got promoted to a high level key

Reply:
I updated the changelog.